extensions/wikia/SpecialDiscussionsLog/SpecialDiscussionsLog.i18n.php

```diff
@@ -11,7 +11,7 @@
 	'discussionslog-view-logs' => 'View Logs',
 	'discussionslog-no-user-match-error' => 'Provided username did not match any user',
 	'discussionslog-ip-invalid-error' => 'Provided IP address is invalid.',
-	'discussionslog-no-mobile-activity-error' => 'No mobile app activity by $1 in the past two weeks!',
+	'discussionslog-no-activity-error' => 'No contributions by $1 in the past two weeks!',
 	'discussionslog-no-ip-activity-error' => 'No recent activity from IP address $1 found.',
```

Comment:
@katanka As discussed offline, better approach is to create a new translation key and then remove the old one. Also we need a LOC ticket.